qutip/bloch.py

```diff
@@ -35,9 +35,13 @@
 
 import os
 
+import matplotlib
```

Comment:
This import should be inside the `try: ... except: ...` block a little futher down. This file should be importable even if matplotlib is not installed (or at least that seems to be the intention).

Reply:
Thanks, I transferred it inside the `try` block..